conf/rest/9.12.0/disk.yaml

```diff
@@ -36,6 +36,8 @@ plugins:
   - LabelAgent:
       value_to_num:
         - new_status outage - - `0` #ok_value is empty value, '-' would be converted to blank while processing.
+      compute_metric:
+        - uptime MULTIPLY stats.power_on_hours 60 60
```

Comment:
We already have one small disk rest plugin, this can be done there as well.

This approach seems changing the definition of `compute_metric`, So need to change doc here: https://github.com/NetApp/harvest/blob/33e2bc610ffa4d9bf2269cc604d8f58e5eee0904/cmd/poller/plugin/README.md#compute_metric